js/components/controlBar.js

```diff
@@ -63,6 +63,22 @@ var ControlBar = React.createClass({
     this.props.controller.toggleFullscreen();
   },
 
+  handleLiveClick: function(evt) {
+    evt.stopPropagation();
+    evt.cancelBubble = true;
+    evt.preventDefault();
+    this.props.controller.seek(this.props.duration);
+  },
+
+  checkLiveCursor: function() {
+    var timeShift = this.props.currentPlayhead - this.props.duration;
+    if (Math.abs(timeShift) < 2) {
+      ReactDOM.findDOMNode(this.refs.LiveButton).style.cursor="default";
```

Comment:
We are trying not to change styling outside of css when possible. Here, for example, instead of changing the style you could change the class name for the element (using ClassNames, see volume as an example) depending on its status. You can do it in populateControlBar, and remove onMouseOver altogether.

    var liveClass = ClassNames({
        "oo-control-bar-item oo-live oo-live-indicator": true,
        "oo-live-nonclickable": Math.abs(this.props.currentPlayhead - this.props.duration) < 2
      });

and then set 'cursor: default' in css for oo-live-nonclickable